appendix-basic65.tex

```diff
@@ -10133,7 +10135,7 @@ \subsection{WPEEK}
 
 \item [Format:]   {\bf WPEEK(}address{\bf)}
 
-\item [Returns:]  The 16-bit word value stored in memory at {\bf address} (low byte) and {\bf address} + 1 (high byte), as an unsigned 16-bit number.
+\item [Returns:]  The 16-bit word value stored in memory at {\bf address} (low byte) and {\bf address + 1} (high byte), as an unsigned 16-bit number.
```

Comment:
I think this just means to use bold for the "address" argument name, so the +1 shouldn't be bolded.